l2geth/core/rollup_fee_test.go

```diff
@@ -23,9 +23,21 @@ func TestCalculateRollupFee(t *testing.T) {
 			data := make([]byte, 0, tt.dataLen)
 			fee := CalculateRollupFee(data, tt.gasUsed, big.NewInt(tt.dataPrice), big.NewInt(tt.executionPrice))
 
-			dataFee := uint64((RollupBaseTxSize + len(data)) * int(tt.dataPrice))
+			var zeros uint64
```

Comment:
This should go in a helper function if the same logic is duplicated in the code and the test